modules/pricing/regions.tf

```diff
@@ -13,6 +13,7 @@ locals {
   # Note: Adding more regions and local zones. Copied from https://github.com/powdahound/ec2instances.info/blob/master/ec2.py
   all_aws_regions = merge({ for k, v in data.aws_region.one : replace(v.description, "Europe", "EU") => k }, {
     "Asia Pacific (Osaka-Local)" = "ap-northeast-3"
+    "Asia Pacific (Jakarta)"     = "ap-southeast-3"
```

Comment:
Same as with this one, I suppose. Jakarta region is a proper AWS region, not a Local Zone like Osaka. Could you revert this one and try it?